Sources/Classes/BiometryService.swift

```diff
@@ -7,18 +7,59 @@
 
 import LocalAuthentication
 
+/// The protocol used to define the specifications necessary for the SDK.
 public protocol BiometryService: AnyObject {
 
+  /// A boolean value stating whether the biometric feature is enabled.
   var isEnabled: Bool { get set }
+
+  /// A boolean value stating whether the biometric feature is available.
   var isAvailable: Bool { get }
+
+  /// Indicates the type of the biometry supported by the device.
   var supportedType: BiometryType { get }
+
+  /// Fallback button title.
   var localizedFallbackTitle: String? { get set }
+
+  /// Cancel button title.
   var localizedCancelTitle: String? { get set }
+
+  /// Allows setting the default localized authentication reason on context.
   var localizedReason: String { get set }
+
+  /// Allows running authentication in non-interactive mode.
   var interactionNotAllowed: Bool { get set }
 
+  /// Authenticates the user when using the application.
+  ///
+  /// @param completion Completeion block that is executed when authentication finishes.
+  ///              success Completion parameter that is success if the authentication has been authenticate successfully.
+  ///              failure Completion parameter that is failed if the authentication has been authenticate failed.
+  ///                    contains error information about the authentication failure.
```

Comment:
A few improvements for avoiding duplicating words in the same sentence 🙏
```suggestion
  ///              success Completion parameter that is success if the authentication has been executed successfully.
  ///              failure Completion parameter that is failed if the authentication has failed.
  ///                    contains error information about the authentication's failure.
```